spec/factories/offsprings.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :offspring do
+    first_name "MyString"
+    last_name "MyString"
+    type ""
```

Comment:
Why do you add type here? It should be populated by Rails
@amaurygonzalez can you review this?